fizz_buzz.rb

```diff
@@ -1,12 +1,6 @@
 #!/usr/bin/env ruby
 
-def divisible_by_3?(number)
-  (number % 3).zero?
-end
-
-def divisible_by_5?(number)
-  (number % 5).zero?
-end
+require_relative "fizz_buzz_class"
```

Comment:
Rename the helper module to `fizzbuzz_helper.rb`

Reply:
Also, it's better if you change the name of the main program,. Something like `display_fizzbuzz.rb`
That way, you can name the class with the fizzbuzz logic `fizzbuzz.rb`.

So you should have three files for this PR. The main program, `fizzbuzz_helper.rb` (helpers module), and `fizzbuzz.rb` (the `Fizzbuzz` class).